.coveragerc

```diff
@@ -3,5 +3,6 @@ exclude_lines =
     pragma: no cover
     def __repr__
     def __str__
+    raise AssertionError
```

Comment:
What's that addition good for?

Reply:
This is required for the change in `rflx/generator/common.py`. Previously we used a `assert False` for a branch which should never be reached. While `pylint` does not accept that no `return` follows after the assert, `mypy` shows now that the `return` cannot be reached. The solution was to replace the assert by a `raise AssertionError`. I think the benefit of adding a unit test which tests that the `AssertionError` is raised for such a case is limited. I also see no other case where it would be sensible to raise an `AssertionError`, and which we wouldn't need to test as a consequence. On the other side, adding a unit test for this case wouldn't need that much effort. So I'm not strictly against reverting that change. What is your opinion?

Reply:
I think we should revert the change and add a test. Just in case we have other uses of `AssertionError` in the future.

Reply:
> On the other side, adding a unit test for this case wouldn't need that much effort.

I was too optimistic. Actually, I see no sensible way to add a test for this case. We would need to create a `Message` containing an invalid type. Therefore, we would need to deactivate the contracts and multiple verification checks in `Message`. I don't think it is worth the effort just for testing a "this should never happen" case. I would propose to use `raise AssertionError` explicitly and exclusively for such cases. Actually, I cannot think of any other situation where I would accept using this exception type.

Reply:
Alternatively, we could also replace `raise AssertionError` by `assert False` (in the code and in coverage exclude). Maybe that is even better. The question is: Does it generally make sense to add unit tests for "this should never happen" cases?

Reply:
> Alternatively, we could also replace `raise AssertionError` by `assert False` (in the code and in coverage exclude). Maybe that is even better.

Yes, I also like it better.

> The question is: Does it generally make sense to add unit tests for "this should never happen" cases?

TBH, I don't know and I found no satisfying answer to that. For now, I suggest we add an exception for `assert False` and don't test them.